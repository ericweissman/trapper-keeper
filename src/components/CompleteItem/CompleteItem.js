import React from 'react'
import PropTypes from 'prop-types';

export const CompleteItem = (props) => {
  return (
    <div className='completed'>
      <p>{props.description}</p>
    </div>
  )
}

export default CompleteItem

CompleteItem.propTypes = {
  description: PropTypes.string,
}
