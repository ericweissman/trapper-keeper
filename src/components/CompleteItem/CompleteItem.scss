.completed {
  display: flex;
  align-items: center;
  padding: 3px 0;
  p {
    padding-left: 10px;
    text-decoration: line-through;
  }
  word-break: break-all;
}