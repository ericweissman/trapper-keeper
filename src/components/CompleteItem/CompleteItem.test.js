import React from 'react'
import { shallow } from 'enzyme'
import CompleteItem from './CompleteItem'

describe('CompleteItem', () => {
  let wrapper
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<CompleteItem />)
    expect(wrapper).toMatchSnapshot()
  })
})