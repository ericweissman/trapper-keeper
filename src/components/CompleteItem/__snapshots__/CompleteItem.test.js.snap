// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompleteItem should have match the correct snapshot 1`] = `
<div
  className="completed"
>
  <p />
</div>
`;
